### DISCUSSION TITLE
Why cant a player destroy in the wilderness?

### DISCUSSION THREAD

**Discussion by TheWeirdCoder** [TownyAdvanced/Towny#6447] [Q&A] [answered]
So im running a server and my beta tester cant destroy in the wilderness?
Any clue? [They do not have a town]

![image](https://user-images.githubusercontent.com/116373326/213805703-4d28f449-d98a-4f23-a516-2bd7fc52cb4c.png)

**Comment by Warriorrrr** [comment, marked as answer]
See https://github.com/TownyAdvanced/Towny/wiki/Frequently-Asked-Questions
